README.md

```diff
@@ -31,32 +45,42 @@ url=<YOUR_JENKINS_INSTANCE_URL>
 query_plugins_info=False
 ```
 
-### Test your template
+### 1.3 Test your template
 ```
 jenkins-jobs --conf /path/to/your/jenkins_jobs.ini test /path/to/your/template.yaml
 ```
 
-### Update your job
+### 1.4 Update your job
 ```
 jenkins-jobs --conf /path/to/your/jenkins_jobs.ini update /path/to/your/template.yaml
 ```
 
-### Generate inline script jobs
+### 1.5 Generate inline script jobs
 ```
 ./scripts/generate_inline_script_pipeline_job -j /path/to/your/template.yaml -o /path/to/your/generated/jobs
 ```
 
-### Configure all jobs and views
+### 1.6 Configure all jobs and views
 ```
 ./scripts/configure_jenkins /path/to/your/jenkins_jobs.ini
 ```
```

Comment:
Once the work to do all this via ansible is merged we will probably want to remove all this stuff from the readme.